apps/microsoft-teams/frontend/src/components/config/ChannelSelectionSupplementalModalContent/ChannelSelectionSupplementalModalContent.tsx

```diff
@@ -0,0 +1,43 @@
+import { appDeepLink, channelSelection } from '@constants/configCopy';
+import { Button, Modal, Paragraph, TextLink } from '@contentful/f36-components';
+import { TeamsChannel, Notification } from '@customTypes/configPage';
+
+interface Props {
+  children: React.ReactNode;
+  onClose: () => void;
+  handleNotificationEdit: (notificationEdit: Partial<Notification>) => void;
+  selectedChannel: TeamsChannel;
+}
+
+const ChannelSelectionSupplementalModalContent = (props: Props) => {
```

Comment:
Optional suggestion: add a jsdocs `@description` to explain what "supplemental" means in this context.  Which if I'm reading the code correctly, means that we're prepending a description/link paragraph before the `{ children }` node?

Reply:
Yeah the naming of the component was super tricky for me here, because we are adding supplemental information to the main modal content, will add some more description here!